scripts/qa_crowbarsetup.sh

```diff
@@ -2007,8 +2007,13 @@ function reboot_nodes_via_ipmi()
                 if ipmitool -H $ip -U root -P $pw power status | grep -q "is off"; then
                     ipmitool -H $ip -U root -P $pw power on
                     sleep $((10 + RANDOM % 15))
+                else
+                    ipmitool -H $ip -U root -P $pw power off
+                    wait_for 5 2 "ipmitool -H $ip -U root -P $pw power status | grep -q 'is off'" "node to power on"
```

Comment:
I did not check for firmware updates.
I can create an infra ticket.
But this needs to be scheduled because it will affect qa2 and qa3 hardware.